testing/test_framework/src/firebase_test_framework.h

```diff
@@ -38,12 +38,12 @@ namespace firebase_test_framework {
 
 // Use this macro to skip an entire test if it is an non UI Test and we are
 // not running it in UItest mode (for example, on UI Test workflow).
-#define TEST_NO_USER_INTERACTION                                            \
-  if (SkipNonUITest()) {                                                    \
-    app_framework::LogInfo("Skipping %s, as it is an Non UI Test.",         \
-                            test_info_->name());                            \
-    GTEST_SKIP();                                                           \
-    return;                                                                 \
+#define TEST_NO_USER_INTERACTION                                    \
```

Comment:
Will we have to set this manually at the start of every test? I wonder if there is a better way, maybe in the FirebaseTest setup / initialization methods.

Reply:
I also feel like it should be named `TEST_REQUIRES_NO_USER_INTERACTION`.

Reply:
Yes. I am aware of this problem. It's annoying, even if we only need to set this Marco to GMA testapp. And that's why I plan to have a separate PR to modify the GMA Integration Tests.

Thanks for the suggestion about `FirebaseTest setup / initialization methods`. I will have a try.

Reply:
And to Jon's point, isn't this implied by the lack of `TEST_REQUIRES_USER_INTERACTION`?

Reply:
I will have more experiments. If the idea works, then I will have another PR.